bitratechecker.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+import sys
+import os
+import glob
+import subprocess
+import json
+import argparse
```

Comment:
Personally I prefer sorted order (nice discussion about that: https://stackoverflow.com/a/20763446).

Reply:
I think current order reflects how these imports were added during development. I don't have a strong opinion about that, if there are reasons for sorting the imports I can do that.